portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,18 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ['I measure about 5 feet 10 inches!','I am really good at MarioKart and Just Dance!',
+    'Some of my favorite movies are Now You See Me and Ratatouille!','I have a 6 month Cane Corso named Mamba!',
+    'I am allergic to cats:(!','One of my favorite shows is Hells Kitchen!'];
```

Comment:
Should have mentioned this earlier as well, sorry, if all of your object parts (attributes? entries? not sure what the correct word is here, sorry) can't fit in a line, then try to break it up into separate parts that make sense. In this case, you could have each fact on its own line. If you had an object with different attributes, then you could put each attribute on it's own line.

I would do
```
const facts = [
    'fact1',
    'fact2',
    'factX'
];
```